src/goto-instrument/contracts/contracts.cpp

```diff
@@ -11,98 +11,28 @@ Date: February 2016
 /// \file
 /// Verify and use annotated invariants and pre/post-conditions
 
-#include "code_contracts.h"
+#include "contracts.h"
+#include "assigns.h"
+#include "memory_predicates.h"
```

Comment:
[nit]

Consider sorting these

Reply:
`clang-format` adds the corresponding `.h` at the top. I added a new line.